ui/public/locales/pt_BR.json

```diff
@@ -1,879 +1,1143 @@
 "label.add.affinity.group": "Adicionar um grupo de afinidade",
-"label.add.baremetal.dhcp.device": "Adiciona Dispositivo DHCP Baremetal",
-"label.add.baremetal.rack.configuration": "Adicionar Configura\u00e7\u00e3o de Rack de Baremetal",
-"label.add.bigswitchbcf.device": "Adicionar BigSwitch BCF Controller",
-"label.add.brocadevcs.device": "Adicionar Brocade Vcs Switch",
+"label.add.baremetal.dhcp.device": "Adicionar dispositivo DHCP baremetal",
+"label.add.baremetal.rack.configuration": "Adicionar configura\u00e7\u00e3o de rack de baremetal",
+"label.add.bigswitchbcf.device": "Adicionar controlador BigSwitch BCF",
+"label.add.brocadevcs.device": "Adicionar switch Brocade Vcs",
 "label.add.by": "Adicionado por",
 "label.add.by.cidr": "Adicionar por CIDR",
-"label.add.by.group": "Adicionar por Grupo",
-"label.add.ciscoasa1000v": "Adicone Recurso",
-"label.add.cluster": "Adicionar Cluster",
+"label.add.by.group": "Adicionar por grupo",
+"label.add.certificate": "Adicionar certificado",
+"label.add.ciscoasa1000v": "Adicionar recurso",
+"label.add.cluster": "Adicionar cluster",
 "label.add.compute.offering": "Adicionar oferta de computa\u00e7\u00e3o",
-"label.add.direct.iprange": "Add Direct Ip Range",
-"label.add.disk.offering": "Adicionar Oferta de Disco",
-"label.add.domain": "Adicionar Dom\u00ednio",
-"label.add.egress.rule": "Adicionar regra egress",
+"label.add.direct.iprange": "Adicionar intervalo de IP direto",
+"label.add.disk.offering": "Adicionar oferta de disco",
+"label.add.domain": "Adicionar dom\u00ednio",
+"label.add.egress.rule": "Adicionar regra de sa\u00edda",
 "label.add.f5.device": "Adicionar dispositivo F5",
-"label.add.firewall": "Adicionar regra de Firewall",
+"label.add.firewall": "Adicionar regra de firewall",
 "label.add.globo.dns": "Adicionar GloboDNS",
 "label.add.gslb": "Adicionar GSLB",
 "label.add.guest.network": "Adicionar rede guest",
-"label.add.host": "Adicionar Host",
-"label.add.ingress.rule": "Adicionar Regra de Entrada",
+"label.add.host": "Adicionar host",
+"label.add.ingress.rule": "Adicionar regra de entrada",
 "label.add.intermediate.certificate": "Adicionar certificado intermedi\u00e1rio",
-"label.add.internal.lb": "Adiciona LB Interno",
-"label.add.ip.range": "Adicionar Range de IP",
-"label.add.isolated.guest.network": "Adiciona Rede Guest Isolada",
-"label.add.isolated.guest.network.with.sourcenat": "Adicionar rede Guest isolada com SourceNat",
-"label.add.isolated.network": "Adiciona Rede Isolada",
-"label.add.l2.guest.network": "Add L2 Guest Network",
-"label.add.ldap.account": "Adicionar Conta LDAP",
-"label.add.list.name": "Nome de Lista ACL",
-"label.add.load.balancer": "Adicionar Load Balance",
-"label.add.more": "Adicionar Mais",
+"label.add.internal.lb": "Adicionar LB interno",
+"label.add.ip.range": "Adicionar intervalo de IPs",
+"label.add.isolated.guest.network": "Adicionar rede guest isolada",
+"label.add.isolated.guest.network.with.sourcenat": "Adicionar rede guest isolada com sourceNat",
```

Comment:
Shouldn't it be `Source NAT` instead of `sourceNat`? considering that NAT is an achronim for Network Address Translation.

 Maybe also SNAT